Content.Server/Hands/Systems/HandsSystem.cs

```diff
@@ -148,11 +148,14 @@ private void HandleEntityRemoved(EntityUid uid, SharedHandsComponent component,
         #endregion
 
         #region pulling
-        private static void HandlePullAttempt(EntityUid uid, HandsComponent component, PullAttemptEvent args)
+        private void HandlePullAttempt(EntityUid uid, HandsComponent component, PullAttemptEvent args)
         {
             if (args.Puller.Owner != uid)
                 return;
 
+            if (TryComp<SharedPullerComponent>(args.Puller.Owner, out var pullerComp) && !pullerComp.NeedsHands)
+                return;
+
             // Cancel pull if all hands full.
             if (!component.IsAnyHandFree())
                 args.Cancelled = true;
```

Comment:
Given that this is already checked in `SharedPullingSystem.CanPull()`, I think attempt event subscription should just be removed altogether.

Reply:
Let me try it real quick

Reply:
Works on my machine with or without needsHands